quesma/queryparser/pancake_model.go

```diff
@@ -37,7 +30,7 @@ type pancakeFillingMetricAggregation struct {
 	metadata model.JsonMap
 }
 
-type pancakeLayerBucketAggregation struct {
+type pancakeModelBucketAggregation struct {
 	name            string          // as originally appeared in Query DSL
```

Comment:
Some fields are common between metric and bucket aggregations. Maybe we could merge them? or embed one type in another

Reply:
I agree, but let me do that in the following PR.